config/.eslintrc.js

```diff
@@ -98,18 +98,18 @@ module.exports = {
         'object': 'it',
         'property': 'skip'
       },
-      {
-        'object': 'it',
-        'property': 'only'
-      },
+      // {
```

Comment:
Reminder not to check in. If this is blocking local testing with `only`, just run the test name directly, e.g. `yarn test:browser` to avoid calling the lint job

Reply:
Yep, will remove ASAP